org.kde.elisa.json

```diff
@@ -0,0 +1,107 @@
+{
+  "id": "org.kde.elisa",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.9",
+  "sdk": "org.kde.Sdk",
+  "command": "elisa",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=wayland",
+    "--filesystem=home:ro",
+    "--socket=pulseaudio",
+    "--device=dri",
+    "--talk-name=org.kde.baloo",
+    "--own-name=org.mpris.MediaPlayer2.elisa",
+    "--env=BALOO_DB_PATH=.local/share/baloo"
+  ],
+  "modules": [
+    {
+      "config-opts": [
+        "-DENABLE_TESTING=OFF",
+        "-DCMAKE_INSTALL_LIBDIR=lib"
+      ],
+      "name": "kdeclarative",
+      "buildsystem": "cmake-ninja",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.kde.org/stable/frameworks/5.44/kdeclarative-5.44.0.tar.xz",
+          "sha256": "40d2108b28c59ca0e486b1a7540d6b46a43b5177e0a40b298e40d9e9ba7acf2f"
+        }
+      ]
+    },
+    {
+      "config-opts": [
+        "-DBUILD_SHARED_LIBS=ON"
+      ],
+      "name": "taglib",
+      "buildsystem": "cmake-ninja",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://taglib.github.io/releases/taglib-1.11.1.tar.gz",
+          "sha256": "b6d1a5a610aae6ff39d93de5efd0fdc787aa9e9dc1e7026fa4c961b26563526b"
+        }
+      ]
+    },
+    {
+      "config-opts": [
+        "-DENABLE_TESTING=OFF",
+        "-DCMAKE_INSTALL_LIBDIR=lib"
+      ],
+      "name": "kfilemetadata",
+      "buildsystem": "cmake-ninja",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.kde.org/stable/frameworks/5.44/kfilemetadata-5.44.0.tar.xz",
+          "sha256": "6f9dd7ab57ff4465736272973a2f064fc68ff5dfbf7f56f8f4c7dcc7016d1d6d"
+        }
+      ]
+    },
+    {
+      "name": "lmdb",
+      "sources": [
+        {
+          "type": "git",
+          "url": "git://github.com/LMDB/lmdb.git",
+          "branch": "LMDB_0.9.18",
+          "commit": "ad8488cfac644d7a289e428ab3c403c859d844cb"
+        }
+      ],
+      "no-autogen": true,
+      "make-install-args": [
+        "prefix=/app"
+      ],
+      "subdir": "libraries/liblmdb"
+    },
+    {
+      "config-opts": [
+        "-DENABLE_TESTING=OFF",
+        "-DCMAKE_INSTALL_LIBDIR=lib",
+        "-DLMDB_DIR=/app"
+      ],
+      "name": "baloo",
+      "buildsystem": "cmake-ninja",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.kde.org/stable/frameworks/5.44/baloo-5.44.0.tar.xz",
+          "sha256": "3eae55b5fa31f695da4145f8505637f6d8d5f77a0472d6f23fd367690ba07d79"
+        }
+      ]
+    },
+    {
+      "name": "elisa",
+      "buildsystem": "cmake-ninja",
```

Comment:
May want to add:
```json
"config-opts": [
   "-DCMAKE_BUILD_TYPE=Release"
]
```

Same for other cmake projects.

Reply:
I have added it.